spec/src/main/asciidoc/required-metrics.adoc

```diff
@@ -234,10 +234,10 @@ Vendors should either use other metrics that are close enough as substitute or n
 |Name| cpu.processCpuLoad
 |DisplayName| Process CPU Load
 |Type| Gauge
-|Unit| Percent
+|Unit| Ratio
```

Comment:
Use `Ratio` as a Prometheus base units defines.

Reply:
I'd suggest we don't change this as it will break existing dashboards.  New metrics we define should use ratio rather than percent, but not sure it's worth people having to rework dashboards (and have dashboards know which version of MP Metrics they are using) for this, when Prometheus tolerates custom units.